src/test/java/seedu/placebook/testutil/AppointmentBuilder.java

```diff
@@ -26,13 +26,26 @@
      * Creates a {@code} AppointmentBuilder with the default details.
      */
     public AppointmentBuilder() {
+        this(true);
+    }
+
+    /**
+     * Creates a {@code} AppointmentBuilder with the two default values.
+     * This is a simple way to create appointments without time conflicts.
+     */
+    public AppointmentBuilder(boolean isFirst) {
```

Comment:
Good idea, just wondering if maybe its better to have an Enum like DEFAULT_TIME, so that its clearer, and maybe then we can add more values so that we can play around with more default values for testing.